webapi/src/test/resources/test-data/ontologyR2RV2/standoffOntologySimple.rdf

```diff
@@ -12,557 +12,557 @@
 </owl:Restriction>
-<owl:Restriction rdf:nodeID="genid-55a441049d76415ba3999f7d47ee542f-b3">
+<owl:Restriction rdf:nodeID="genid-0e6b49fe8a374047959ae8a349b1a4de-b3">
 	<owl:maxCardinality rdf:datatype="http://www.w3.org/2001/XMLSchema#integer">1</owl:maxCardinality>
 	<owl:onProperty rdf:resource="http://api.knora.org/ontology/knora-api/simple/v2#standoffTagHasOriginalXMLID"/>
 </owl:Restriction>
-<owl:Restriction rdf:nodeID="genid-55a441049d76415ba3999f7d47ee542f-b4">
+<owl:Restriction rdf:nodeID="genid-0e6b49fe8a374047959ae8a349b1a4de-b4">
 	<owl:cardinality rdf:datatype="http://www.w3.org/2001/XMLSchema#integer">1</owl:cardinality>
 	<owl:onProperty rdf:resource="http://api.knora.org/ontology/knora-api/simple/v2#standoffTagHasStart"/>
 </owl:Restriction>
-<owl:Restriction rdf:nodeID="genid-55a441049d76415ba3999f7d47ee542f-b5">
+<owl:Restriction rdf:nodeID="genid-0e6b49fe8a374047959ae8a349b1a4de-b5">
 	<owl:cardinality rdf:datatype="http://www.w3.org/2001/XMLSchema#integer">1</owl:cardinality>
 	<owl:onProperty rdf:resource="http://api.knora.org/ontology/knora-api/simple/v2#standoffTagHasStartIndex"/>
 </owl:Restriction>
-<owl:Restriction rdf:nodeID="genid-55a441049d76415ba3999f7d47ee542f-b6">
+<owl:Restriction rdf:nodeID="genid-0e6b49fe8a374047959ae8a349b1a4de-b6">
 	<owl:maxCardinality rdf:datatype="http://www.w3.org/2001/XMLSchema#integer">1</owl:maxCardinality>
 	<owl:onProperty rdf:resource="http://api.knora.org/ontology/knora-api/simple/v2#standoffTagHasStartParent"/>
 </owl:Restriction>
-<owl:Restriction rdf:nodeID="genid-55a441049d76415ba3999f7d47ee542f-b7">
+<owl:Restriction rdf:nodeID="genid-0e6b49fe8a374047959ae8a349b1a4de-b7">
 	<owl:cardinality rdf:datatype="http://www.w3.org/2001/XMLSchema#integer">1</owl:cardinality>
 	<owl:onProperty rdf:resource="http://api.knora.org/ontology/knora-api/simple/v2#standoffTagHasUUID"/>
 </owl:Restriction>
 <owl:Class rdf:about="http://api.knora.org/ontology/standoff/simple/v2#StandoffBoldTag">
 	<rdfs:comment rdf:datatype="http://www.w3.org/2001/XMLSchema#string">Represents bold text in a TextValue</rdfs:comment>
 	<rdfs:subClassOf rdf:resource="http://api.knora.org/ontology/standoff/simple/v2#StandoffVisualTag"/>
-	<rdfs:subClassOf rdf:nodeID="genid-55a441049d76415ba3999f7d47ee542f-b8"/>
-	<rdfs:subClassOf rdf:nodeID="genid-55a441049d76415ba3999f7d47ee542f-b9"/>
-	<rdfs:subClassOf rdf:nodeID="genid-55a441049d76415ba3999f7d47ee542f-b10"/>
-	<rdfs:subClassOf rdf:nodeID="genid-55a441049d76415ba3999f7d47ee542f-b11"/>
-	<rdfs:subClassOf rdf:nodeID="genid-55a441049d76415ba3999f7d47ee542f-b12"/>
-	<rdfs:subClassOf rdf:nodeID="genid-55a441049d76415ba3999f7d47ee542f-b13"/>
-	<rdfs:subClassOf rdf:nodeID="genid-55a441049d76415ba3999f7d47ee542f-b14"/>
-	<rdfs:subClassOf rdf:nodeID="genid-55a441049d76415ba3999f7d47ee542f-b15"/>
+	<rdfs:subClassOf rdf:nodeID="genid-0e6b49fe8a374047959ae8a349b1a4de-b8"/>
```

Comment:
Are these blank node Iris?

Reply:
Yes, for cardinalities.

Reply:
But aren't those different for every request you make?

Reply:
Yes. Every time you convert a JSON-LD document containing blank nodes to RDF/XML, you get different randomly generated blank node IDs. Semantically it makes no difference.

Reply:
I just wondered how the tests work. I assume the blank node Iris are just ignored, right?

Reply:
The tests use RDF4J Rio to parse the RDF/XML into an `org.eclipse.rdf4j.model.Model`. If you compare two `Model` objects that contain blank nodes using the `==` operator, the `Model` class understands that the blank node IDs aren't significant. It just compares their contents, not their IDs.

Reply:
Ok, got it. Thanks!